source/transactions/tests/README.rst

```diff
@@ -64,12 +70,15 @@ Each YAML file has the following keys:
 Use as integration tests
 ========================
 
-Run a MongoDB replica set with a primary, two secondaries, and an arbiter,
-server version 4.0 or later. (Including two secondaries ensures that transaction
-pinning works properly. Include an arbiter to ensure no new bugs have been
-introduced related to arbiters.)
+Run a MongoDB replica set with a primary, a secondary, and an arbiter,
+server version 4.0 or later. (Including a secondary ensures that server
+selection in a transaction works properly. Including an arbiter helps ensure
+that no new bugs have been introduced related to arbiters.)
+
+Load each YAML (or JSON) file using a Canonical Extended JSON parser.
+The parser MUST preserve the order of keys in dictionaries.
```

Comment:
Does anyone see any problem with this requirement? Are there any languages that cannot preserve the order of keys when parsing JSON/YAML?

Reply:
Maybe Node?  I'm not sure.  If it's a problem we could pull the command name/value into a separate field from the rest of the command.

Reply:
Okay I added a "command_name" as a hint for drivers that can't preserve key order.

Reply:
thanks!